src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java

```diff
@@ -151,6 +150,11 @@ protected void connect() throws SQLException {
 		log.debug("Connection opened");
 	}
 
+	/**
+	 * Returns the version of the firebolt backend the connection is established to
+	 */
+	public abstract int getInfraVersion();
+
```

Comment:
Do we use it anywhere in our code? If not, we can probably remove infra version altogether. Firebolt 2.0 used to have account v1 and v2 that had different metadata capabilities. Since we only have accounts v2 now there's no need to keep track of the version.

Reply:
Ye, I get that this is a base. I was wondering if we use this method anywhere else, but I see a couple of places where we use it to determine if we're running on Firebolt 2.0 or 1.0.